ur_description/urdf/ur5_robot.urdf.xacro

```diff
@@ -5,6 +5,27 @@
   <!-- ur5 -->
   <xacro:include filename="$(find ur_description)/urdf/ur5.urdf.xacro" />
 
+  <!-- ros2 control argument-->
+  <xacro:arg name="use_ros2_control" default="false"/>
```

Comment:
I think you should set this to **true** because there's no alternative that I'm aware of